### DISCUSSION TITLE
crypt15 decrypt

### DISCUSSION THREAD

**Discussion by rahbadev** [ElDavoo/wa-crypt-tools#148] [General]
After numerous attempts, 0 messages were recovered
This was months ago
Now I set up WhatsApp again, deleted all my conversations, de-encrypted the backup, and changed the email for the backup.

After a while, I remembered the password for my previous version, so I set up WhatsApp and tried to retrieve conversations, and when I entered the password, it said to me, “Your account is not linked to a password. Try the 64-digit encryption key.”

What I understood from your previous posts is that the company stores this key on its servers and this key can be recovered by entering the password

I have questions I want to solve:
1. After changing the email and canceling the encryption password for the copy, will all my encryption keys be erased?
2. Is my encryption key fixed and does not change even if I change the email or device or cancel the backup encryption?

**Comment by ElDavoo** [comment]
> 1. After changing the email and canceling the encryption password for the copy, will all my encryption keys be erased?

I guess...

> 2. Is my encryption key fixed and does not change even if I change the email or device or cancel the backup encryption

I think if you disable and re-enable the encryption the key should change.

**Reply by rahbadev** [reply]
Thanks, for your reply, So what I understand is that it is impossible for me to restore my backup Since I have the password and I have the files and databases of the full backup, I mean the (Whatsapp)folder

Or is there a solution to this so that I can restore my chats, or at least see them

**Reply by ElDavoo** [reply]
I don't know, I don't think so.

**Reply by rahbadev** [reply]
mmmmm, Thank you for your help
